### DISCUSSION TITLE
Writing input files

### DISCUSSION THREAD

**Discussion by emiliakrakowska** [modflowpy/flopy#1460] [Q&A]
Hi, I am trying to run an existing MF-NWT model on Flopy. Although "nam" file is successfully imported, writing input files crashes always on MNWI. Can you suggest what can I do?

MNWI data:
80 82 81

**Comment by wpbonelli** [comment]
Hi @emiliakrakowska, we'll need more detail to make a suggestion. Can you share the error message you're seeing and describe steps to reproduce?
